input/pagecontent/terminology.md

```diff
@@ -7,93 +7,119 @@ As well as these included definitions this specification typically references [N
+|[Reason Vaccine Administered](https://healthterminologies.gov.au/fhir/ValueSet/reason-vaccine-administered-1)|[AU Base Immunisation](StructureDefinition-au-immunization.html)|NCTS|
+|[Reason Vaccine Not Administered](https://healthterminologies.gov.au/fhir/ValueSet/reason-vaccine-not-administered-3)|[AU Base Immunisation](StructureDefinition-au-immunization.html)|NCTS|
+|[Related Person Relationship Type](https://healthterminologies.gov.au/fhir/ValueSet/related-person-relationship-type-1)|[AU Base Related Person](StructureDefinition-au-relatedperson.html)|NCTS|
+|[Route of Administration](https://healthterminologies.gov.au/fhir/ValueSet/route-of-administration-1)|[AU Base Allergy Intolerance](StructureDefinition-au-allergyintolerance.html), [AU Base Dosage](StructureDefinition-au-dosage.html), [AU Base Medication Administration](StructureDefinition-au-medicationadministration.html)|NCTS|
+|[Separation Mode](https://healthterminologies.gov.au/fhir/ValueSet/separation-mode-1)|[AU Base Encounter](StructureDefinition-au-encounter.html)|NCTS|
+|[Service Provision Conditions Australian Concepts](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-service-provision-conditions.html)|[AU Base Healthcare Service](StructureDefinition-au-healthcareservice.html)|NCTS|
+|[Service Type](https://healthterminologies.gov.au/fhir/ValueSet/service-type-1)|[AU Base Encounter](StructureDefinition-au-encounter.html), [AU Base Healthcare Service](StructureDefinition-au-healthcareservice.html)|NCTS|
+|[ServiceDeliveryLocationRoleType - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-au-v3-ServiceDeliveryLocationRoleType-extended.html)|[AU Base Location](StructureDefinition-au-location.html)|AU Base|
+|[Specimen Collection Procedure](https://healthterminologies.gov.au/fhir/ValueSet/specimen-collection-procedure-1)|[AU Base Specimen](StructureDefinition-au-specimen.html)|NCTS|
+|[Specimen Container Type](https://healthterminologies.gov.au/fhir/ValueSet/specimen-container-type-1)|[AU Base Specimen](StructureDefinition-au-specimen.html)|NCTS|
+|[Specimen Type](https://healthterminologies.gov.au/fhir/ValueSet/specimen-type-1)|[AU Base Specimen](StructureDefinition-au-specimen.html)|NCTS|
+|[Substance](https://healthterminologies.gov.au/fhir/ValueSet/substance-1)|[AU Base Substance](StructureDefinition-au-substance.html)|NCTS|
+|[Vaccination Target Disease](https://healthterminologies.gov.au/fhir/ValueSet/vaccination-target-disease-1)|[AU Base Immunisation](StructureDefinition-au-immunization.html)|NCTS|
+{:.grid}
 
 
 ### Code Systems
 
 See the [FHIR terminology section]({{site.data.fhir.path}}terminologies-systems.html) for a complete discussion on code systems and a list of codes system names used in FHIR.  Most code systems used in this guide are defined in the base FHIR specification. The following code systems are unique to this guide and not listed in the base FHIR specification.
```

Comment:
This paragraph doesn't work. As we are making this improvement. Let's replace 'The following code systems are unique to this guide and not listed in the base FHIR specification' with 'The following code systems are used in the above value sets.

Reply:
Replaced, thanks.

Reply:
Replaced, thanks.